src/main/java/no/nav/tag/kontakt/oss/salesforce/SalesforceKlient.java

```diff
@@ -0,0 +1,88 @@
+package no.nav.tag.kontakt.oss.salesforce;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.tag.kontakt.oss.Kontaktskjema;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Slf4j
+@Component
+public class SalesforceKlient {
+    private final static ObjectMapper objectMapper = new ObjectMapper();
+
+    private final RestTemplate restTemplate;
+
+    private final String authUrl;
+    private final String apiUrl;
+    private final String username;
+    private final String password;
+    private final String clientId;
+    private final String clientSecret;
+
+    public SalesforceKlient(
+            @Qualifier("restTemplate") RestTemplate restTemplate,
```

Comment:
Hvis den andre restTemplate heter `gsakRestTemplate`, kanskje kunne denne hete `salesForceRestTemplate`

Reply:
gsakRestTemplate har en config som er unik for gsak, men denne restTemplaten er bare helt vanilla uten noe mer. Så hvis man skal lage et nytt rest-kall, vil jeg at denne vanlige skal benyttes